src/sensors/mpu6050sensor.cpp

```diff
@@ -35,6 +35,8 @@
 #include "calibration.h"
 #include "GlobalVars.h"
 
+#define MPU6050_2G_TO_MPS2 (1.0f / 16384.0f) * 9.80665f
```

Comment:
I'd suggest using the formula I introduced in my scaling PR
https://github.com/SlimeVR/SlimeVR-Tracker-ESP/pull/182/files#diff-2703084ecf444c2f3b63f844e0329aa9ea317153612daaee625b97de26896f5cR38

Reply:
I assumed it was already computed at compile time I'll change that